### PR TITLE
CDAP-15128: Improve BigQuery package structure

### PR DESCRIPTION
**Pull request by aielchiieva**
This PR creates three the sub-packages in `co.cask.gcp.bigquery` package: 
- source
- sink
- util

and moves appropriate classes into these packages.

### CODE REVIEW

File: src/main/java/co/cask/gcp/bigquery/util/BigQueryUtil.java [hunk 1]
```diff
@@ -45,7 +47,7 @@
 /**
  * Common Util class for big query plugins such as {@link BigQuerySource} and {@link BigQuerySink}
  */
-final class BigQueryUtils {
+public final class BigQueryUtil {
```
Comment (jayfjin) [inline src/main/java/co/cask/gcp/bigquery/util/BigQueryUtil.java:50]: why the change from BigQueryUtils to BigQueryUtil? I think most other places use the plural form

**Review by jayfjin — COMMENTED** [review]
Just to confirm, these will still be in the same jar?

File: src/main/java/co/cask/gcp/bigquery/util/BigQueryUtil.java (hunk 1 above)
Comment (aielchiieva) [reply src/main/java/co/cask/gcp/bigquery/util/BigQueryUtil.java:50]: Looked through the `google-cloud` project, we have the following:
`common` package: FileSetUtil (actually unused in the project), GCPUtils
`datastore` package: DatastoreUtil + two util packages in source and sink sub-packages
`spanner` package: SpannerUtil

Looks like `util` is slightly more used than `utils`. Personally I also prefer `util` naming.
If you want, I can also rename `GCPUtils` to `GCPUtil` for consistency.

**Comment by aielchiieva** [comment]
@jayfjin 
> Just to confirm, these will still be in the same jar?

Yes, this does not affect jar packaging.

File: src/main/java/co/cask/gcp/bigquery/util/BigQueryUtil.java (hunk 1 above)
Comment (jayfjin) [reply src/main/java/co/cask/gcp/bigquery/util/BigQueryUtil.java:50]: I see, seems like a different naming convention inside google-cloud vs cdap. I'm ok with either then
